cdap-app-fabric/src/main/java/co/cask/cdap/app/runtime/workflow/WorkflowStatus.java

```diff
@@ -25,12 +25,12 @@
 
   private  Service.State state;
   private  WorkflowActionSpecification currentAction;
```

Comment:
Is it still valid to use spec here? I thought fork node type doesn't have action?